packages/cardano-services/test/Utxo/UtxoHttpService.test.ts

```diff
@@ -12,13 +12,15 @@ import { OgmiosCardanoNode } from '@cardano-sdk/ogmios';
 import { Pool } from 'pg';
 import { getPort } from 'get-port-please';
 import { healthCheckResponseMock, mockCardanoNode } from '../../../core/test/CardanoNode/mocks';
+import { loadApiVersion } from '../util';
 import { logger } from '@cardano-sdk/util-dev';
 import axios from 'axios';
 
 const APPLICATION_JSON = 'application/json';
 const APPLICATION_CBOR = 'application/cbor';
 const UNSUPPORTED_MEDIA_STRING = 'Request failed with status code 415';
 const BAD_REQUEST_STRING = 'Request failed with status code 400';
+const version = { api: loadApiVersion(), software: '0.0.0' };
```

Comment:
This could be reused from a test `util.ts` as it's the same for all providers

Reply:
https://github.com/input-output-hk/cardano-js-sdk/pull/634/commits/36fa5abb7db75b5a8fd8eaeded9997e731fa20e0